R/carehomes.R

```diff
@@ -404,6 +404,12 @@ carehomes_parameters <- function(start_date, region,
   ## Proportion of population with covid-like symptoms without covid
   ret$p_NC <- p_NC
 
+  ## relative transmissibility of various I compartments
+  ret$I_A_transmission <- 0.363
```

Comment:
where has this number come from? can we add a reference as we will forget?

Reply:
This number has been calculated by us - which will be described in the revised paper.